app/routes/interestgroups/components/InterestGroupMemberList.js

```diff
@@ -11,24 +11,65 @@ import Icon from 'app/components/Icon';
 import type { User, GroupMembership } from 'app/models';
 import sortBy from 'lodash/sortBy';
 
-const iconName = (role: string) => {
+const ExtraInfo = ({ user, role }: { user: User, role: string }) => {
+  if (role === 'leader' || role === 'co_leader') {
+    var _title: string;
+    var _roleStyle;
+    switch (role) {
+      case 'leader':
+        _title = '(Leder)';
+        _roleStyle = styles.leader;
+
+        break;
+
+      case 'co_leader':
+        _title = '(Nestleder)';
+        _roleStyle = styles.coleader;
+        break;
+
+      default:
+        break;
+    }
+
+    const title = _title;
+    const roleStyle = _roleStyle;
+
+    return (
+      <span>
+        <span className={roleStyle}>{user.fullName} </span>
+        <span className={styles.suffix}>{title}</span>
+      </span>
+    );
+  }
+  return <span>{user.fullName}</span>;
```

Comment:
It's not so good practice to "encapsulate" much good inside if-statements. Better to check for the opposite and return something earlier. See https://www.geeksforgeeks.org/writing-clean-else-statements/ for a better explanation. This isn't so bad though, but it's nice to build good habits early 🙌🏼